src/quo/components/wallet/address_text/schema.cljs

```diff
@@ -10,5 +10,11 @@
       [:format {:optional true} [:enum :short :long]]
       [:theme :schema.common/theme]
       [:networks {:optional true}
-       [:maybe [:sequential [:map [:network-name :keyword] [:short-name :string]]]]]]]]
+       [:maybe [:sequential [:map [:network-name :keyword] [:short-name :string]]]]]
+      [:full-address? {:optional true} [:maybe :boolean]]
+      [:size {:optional true} [:maybe :keyword]] ;; TODO: it's text schema and should be implemented in
+                                                 ;; its component and imported here
+      [:weight {:optional true} [:maybe :keyword]] ;; TODO: it's text schema and should be implemented in
+                                                   ;; its component and imported here
+     ]]]
```

Comment:
they look lonely)

Reply:
What a nice comment :D. Yes it's because of the comments. If I move them above, they will be part of the comment

Reply:
Done